internal/commands/app/create_inputs_test.go

```diff
@@ -209,14 +210,52 @@ func TestAppCreateInputsResolveDirectory(t *testing.T) {
 		assert.Equal(t, "new-app", inputs.LocalPath)
 	})
 
-	t.Run("should error when path specified is another realm app", func(t *testing.T) {
+	t.Run("should create default local directory name when ui is set to auto confirm", func(t *testing.T) {
 		profile, teardown := mock.NewProfileFromTmpDir(t, "app_create_test")
 		defer teardown()
 
-		specifiedDir := "test-dir"
-		inputs := createInputs{LocalPath: specifiedDir}
+		_, _, ui, err := mock.NewVT10XConsoleWithOptions(
+			mock.UIOptions{AutoConfirm: true},
+			new(bytes.Buffer))
+		assert.Nil(t, err)
+
+		testAppName := "test-app"
+		newDefaultName := "test-app-1"
+		inputs := createInputs{newAppInputs: newAppInputs{Name: testAppName}}
+
+		err = os.Mkdir(path.Join(profile.WorkingDirectory, testAppName), os.ModePerm)
+		assert.Nil(t, err)
+
+		dir, err := inputs.resolveLocalPath(ui, profile.WorkingDirectory)
+		assert.Nil(t, err)
+		assert.Equal(t, path.Join(profile.WorkingDirectory, newDefaultName), dir)
+		assert.Equal(t, testAppName, inputs.LocalPath)
```

Comment:
I guess I would have expected `LocalPath` to reflect the `dir` we returned from `resolveLocalPath`, so my question is do we need this assert?  If we don't need to mutate `LocalPath` for this command to work then we probably don't need to test its behavior for now